packages/donor/src/screens/UpcommingDonation/UpcomingDonationScreen.tsx

```diff
@@ -15,19 +15,75 @@ import Cancellation from "../../assets/images/cancelation.svg";
 import Whatsapp from "../../assets/images/whatsup-color-big.svg";
 import TrashIcon from "../../assets/icons/trash.svg";
 import UpcomingDonationInfo from "./UpcomingDonationInfo";
+import makeUrls, { TCalendarEvent } from "add-event-to-calendar";
+import ICalendarLink from "react-icalendar-link";
+import Menu from "@material-ui/core/Menu";
```

Comment:
`Donor` is not dependent on Material UI, so you can't import it here.

Please ask Merav or Matar how this should look in the client, and let's implement this without using Material UI.

Could we use a popup for this?

Reply:
Ok, I removed Material UI usage